src/itype.h

```diff
@@ -923,6 +924,9 @@ struct itype {
         /** Weight difference with the part it replaces for mods */
         units::mass integral_weight = -1_gram;
 
+        // information related to being able to store things inside the item.
+        std::vector<pocket_data> pocket_data;
```

Comment:
Regarding the error thrown by the build bot:

`pocket_data` now has two meanings: the local member, and the type. Either rename the member, or use a fully qualified type name:

```suggestion
        std::vector<::pocket_data> pocket_data;
```
(This makes it unambiguous: `pocket_data` is the member, and `::pocket_data` is the global type.)